electives/mocha-testing.md

```diff
@@ -122,6 +122,28 @@ Mocha String Test
     at Context.<anonymous> (test/myTest.js:5:16)
 ```
 - When calling `assert.equal(a, b)` mocha expects the values to be equivalent, or else the test will fail.
+- If you need to test asynchronous code, Mocha makes it very simple using callbacks. All you need to do is pass a callback function (usually named `done`) to `it` function and invoke the callback function after your test is complete. 
+```javascript
+var assert = require('assert');
+
+function asyncFunction(stringToTest, callback) {
+   setTimeout(function() {
```

Comment:
Can you warn them that this is not best practice and why?

Reply:
Done